packages/apps/human-app/server/src/modules/job-assignment/job-assignment.service.ts

```diff
@@ -0,0 +1,79 @@
+import { HttpException, HttpStatus, Injectable } from '@nestjs/common';
+import { lastValueFrom } from 'rxjs';
+import {
+  JobsAssignmentParamsCommand,
+  JobsAssignmentParamsData,
+  JobAssignmentResponse,
+  JobAssignmentCommand,
+  JobAssignmentData,
+  JobsAssignmentResponse,
+} from './interfaces/job-assignment.interface';
+import { HttpService } from '@nestjs/axios';
+import { InjectMapper } from '@automapper/nestjs';
+import { Mapper } from '@automapper/core';
+
+@Injectable()
+export class JobAssignmentService {
+  constructor(
+    public httpService: HttpService,
+    @InjectMapper() private readonly mapper: Mapper,
+  ) {}
+
+  async processJobAssignment(
+    url: string,
+    jobAssignmentCommand: JobAssignmentCommand,
+  ): Promise<JobAssignmentResponse> {
+    const jobAssignmentData = this.mapper.map(
+      jobAssignmentCommand,
+      JobAssignmentCommand,
+      JobAssignmentData,
+    );
+    try {
+      const options = {
+        method: 'POST',
+        url: `${url}/assignment`,
+        data: jobAssignmentData,
+      };
+      const response = await lastValueFrom(this.httpService.request(options));
+      return response.data;
+    } catch (error) {
+      if (error.response) {
+        throw new HttpException(error.response.data, error.response.status);
+      } else {
+        throw new HttpException(
+          'Internal Server Error',
+          HttpStatus.INTERNAL_SERVER_ERROR,
+        );
+      }
+    }
+  }
+
+  async processGettingAssignedJobs(
+    url: string,
+    jobsAssignmentParamsCommand: JobsAssignmentParamsCommand,
+  ): Promise<JobsAssignmentResponse> {
+    const jobsAssignmentParamsData = this.mapper.map(
+      jobsAssignmentParamsCommand,
+      JobsAssignmentParamsCommand,
+      JobsAssignmentParamsData,
```

Comment:
I don't understand usage of this: 
 In my opinion - command should be ready to send request, this is datatype used in the service layer, data should be datatype returned to the controller - so I would remove this additional mapping, and changed returned values from ...Response to ...Data

Reply:
`~Data` type is used for external API's for request purposes and `~Response` data type for their returns (look at `user-operator` and `user-worker` modules)